at_onboarding_cli/lib/src/at_onboarding_service.dart

```diff
@@ -0,0 +1,184 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:at_client/at_client.dart';
+import 'package:at_commons/at_builders.dart';
+import 'package:at_commons/at_commons.dart';
+import 'package:at_lookup/at_lookup.dart';
+import 'package:at_onboarding_cli/at_onboarding_cli.dart';
+import 'package:at_utils/at_logger.dart';
+import 'package:crypton/crypton.dart';
+import 'package:encrypt/encrypt.dart';
+import 'package:zxing2/qrcode.dart';
+import 'package:image/image.dart';
+
+///class containing service that can onboard/authenticate @signs
+class OnboardingService {
```

Comment:
create an abstract class OnboardingService with onboard method..Rename this class to OnboardingServiceImpl